### PR TITLE
Migrate readme yaml

### PR DESCRIPTION
**Pull request by solairerove**
## what
- updated `README.yaml` file
- add tags and categories
- rebuild `README.md` file
- regenerate readme md
- add description key in outputs values
- add makefile
- ignore harness
- prepare to open discussions on review

## why
- need to add categories and tags so we can pull them into the documentation

### CODE REVIEW

File: README.yaml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+description: |-
+  terraform-template-user-data-cloud
+
+# How to use this project
+# usage: |-
+#   Include this module in your existing terraform code:
+
+#   ```hcl
+#   module "user_data_nfs" {
+#     source = "git::https://github.com/cloudposse/terraform-template-user-data-assests.git?ref=master"
```
Comment (osterman) [inline README.yaml:79]: Wrong module referenced

File: README.yaml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    image: "https://img.shields.io/github/release/cloudposse/terraform-template-user-data-cloud.svg"
+    url: "https://github.com/cloudposse/terraform-template-user-data-cloud/releases/latest"
+  - name: "Slack Community"
+    image: "https://slack.cloudposse.com/badge.svg"
+    url: "https://slack.cloudposse.com"
+
+related:
+  - name: "terraform-template-user-data-assests"
+    description: "terraform-template-user-data-assests"
+    url: "https://travis-ci.org/cloudposse/terraform-template-user-data-assests"
```
Comment (osterman) [inline README.yaml]: Wrong base url =) Should be github.

Comment (osterman) [reply README.yaml]: Correct the other urls too.

File: docs/terraform.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+| namespace |  | string | - | yes |
+| region | AWS region to manage instances in | string | - | yes |
+| stage |  | string | - | yes |
+
+## Outputs
+
+| Name | Description |
+|------|-------------|
+| policy_arn |  |
+| user_data |  |
```
Comment (osterman) [inline docs/terraform.md]: Update the output of the module. Set a description of:
```
User data script that should be executed on startup
```

File: README.yaml (hunk 2 above)
Comment (solairerove) [reply README.yaml]: done

File: docs/terraform.md (hunk 3 above)
Comment (solairerove) [reply docs/terraform.md]: done

File: README.yaml (hunk 1 above)
Comment (solairerove) [reply README.yaml:79]: i need help with example here

**Review by solairerove — COMMENTED** [review]
done

**Comment by solairerove** [comment]
i ran `terraform fmt` but it is still failing

**Pull request closed without merging**